### PR TITLE
Adjust the submit button in add linix use case

### PR DESCRIPTION
**Pull request by LinconOssa**
Does the job of returning me. After clicking “Finish”, the page should redirect to /privileges/linixUseCase

### CODE REVIEW

File: src/pages/privileges/outlets/linixUseCase/adding-linix-use-case/index.tsx [hunk 1]
```diff
@@ -289,6 +290,13 @@ function AddingLinixUseCase() {
     setShowModal((prevShowModal) => !prevShowModal);
   };
 
+  const navigate = useNavigate();
+  const handleFinishForm = () => {
+    saveLinixUseCase(formData);
+    handleToggleModal();
```
Comment (carin91) [inline src/pages/privileges/outlets/linixUseCase/adding-linix-use-case/index.tsx:296]: The expected behavior of the "enviar" button is this, the click triggers a 2 steps process:
(1) A success/error message is rendered
image
(2) The app navigates to privileges/linixUseCase

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/privileges/outlets/linixUseCase/adding-linix-use-case/index.tsx [hunk 2]
```diff
@@ -13,8 +13,9 @@ import {
   IFormAddLinixUseCaseRef,
   IClientServerButton,
 } from "./types";
-import { addLinixUseCaseStepsRules } from "./utils";
+import { addLinixUseCaseStepsRules, saveLinixUseCase } from "./utils";
 import { IAssignmentFormEntry } from "../../users/types/forms.types";
+import { useNavigate } from "react-router-dom";
```
Comment (carin91) [inline src/pages/privileges/outlets/linixUseCase/adding-linix-use-case/index.tsx:18]: Adjust the imports.

File: src/pages/privileges/outlets/linixUseCase/adding-linix-use-case/interface.tsx [hunk 3]
```diff
@@ -31,7 +31,7 @@ import { buttonOptionsMock } from "@src/mocks/privileges/button/buttonOptionsMoc
```
Comment (carin91) [inline src/pages/privileges/outlets/linixUseCase/adding-linix-use-case/interface.tsx:31]: Adjust the imports

**Review by carin91 — CHANGES_REQUESTED** [review]

File: src/pages/privileges/outlets/linixUseCase/adding-linix-use-case/index.tsx (hunk 2 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/linixUseCase/adding-linix-use-case/index.tsx]: resolved

File: src/pages/privileges/outlets/linixUseCase/adding-linix-use-case/interface.tsx (hunk 3 above)
Comment (LinconOssa) [reply src/pages/privileges/outlets/linixUseCase/adding-linix-use-case/interface.tsx:32]: resolved

**Review by carin91 — APPROVED** [review]

**Pull request merged**